test/e2e/framework/test_context.go

```diff
@@ -245,6 +246,7 @@ func RegisterClusterFlags() {
 	flag.StringVar(&cloudConfig.Zone, "gce-zone", "", "GCE zone being used, if applicable")
 	flag.StringVar(&cloudConfig.Region, "gce-region", "", "GCE region being used, if applicable")
 	flag.BoolVar(&cloudConfig.MultiZone, "gce-multizone", false, "If true, start GCE cloud provider with multizone support.")
+	flag.BoolVar(&cloudConfig.MultiMaster "gce-multimaster", false, "If true, multi-master cluster is being used.")
```

Comment:
Maybe this should be gke-multimaster? WDYT?
Any better suggestion?

Reply:
Could you mention GKE in the flag description to indicate it's used by it?

Reply:
Done